common/app/model/dotcomrendering/Trail.scala

```diff
@@ -0,0 +1,167 @@
+      metadata = maybeContent.metadata
+      pillar <- metadata.pillar
+      url <- properties.webUrl
+      headline = contentCard.header.headline
+      isLiveBlog = properties.isLiveBlog
+      showByline = properties.showByline
+      webPublicationDate <- contentCard.webPublicationDate.map(x => x.toDateTime().toString())
+      shortUrl <- contentCard.shortUrl
+    } yield Trail(
+      url = url,
+      linkText = "",
+      showByline = showByline,
+      byline = contentCard.byline.map(x => x.get),
+      image = maybeContent.trail.thumbnailPath,
+      carouselImages = getImageSources(maybeContent.trail.trailPicture),
+      ageWarning = None,
+      isLiveBlog = isLiveBlog,
+      pillar = TrailUtils.normalisePillar(Some(pillar)),
+      designType = metadata.designType.toString,
+      format = metadata.format.getOrElse(ContentFormat.defaultContentFormat),
+      webPublicationDate = webPublicationDate,
+      headline = headline,
+      mediaType = contentCard.mediaType.map(x => x.toString),
+      shortUrl = shortUrl,
+      kickerText = contentCard.header.kicker.flatMap(_.properties.kickerText),
+      starRating = contentCard.starRating,
+      avatarUrl = contentCardToAvatarUrl(contentCard),
+      branding = contentCard.branding,
+    )
+  }
+
+  def pressedContentToTrail(content: PressedContent)(implicit
+      request: RequestHeader,
+  ): Trail = {
+
+    def pillarToString(pillar: Pillar): String = {
+      pillar.toString.toLowerCase() match {
+        case "arts" => "culture"
+        case other  => other
+      }
+    }
+    Trail(
+      url = LinkTo(content.header.url),
+      linkText = RemoveOuterParaHtml(content.properties.linkText.getOrElse(content.header.headline)).body,
+      showByline = content.properties.showByline,
+      byline = content.properties.byline,
+      image = content.trailPicture.flatMap(ImgSrc.getFallbackUrl),
+      carouselImages = getImageSources(content.trailPicture),
+      ageWarning = content.ageWarning,
+      isLiveBlog = content.properties.isLiveBlog,
+      pillar = content.maybePillar.map(pillarToString).getOrElse("news"),
+      designType = content.properties.maybeContent.map(_.metadata.designType).getOrElse(Article).toString,
+      format = content.format.getOrElse(ContentFormat.defaultContentFormat),
+      webPublicationDate = content.webPublicationDate.withZone(DateTimeZone.UTC).toString,
+      headline = content.header.headline,
+      mediaType = content.card.mediaType.map(_.toString()),
+      shortUrl = content.card.shortUrl,
+      kickerText = content.header.kicker.flatMap(_.properties.kickerText),
+      starRating = content.card.starRating,
+      avatarUrl = None,
+      branding = content.branding(Edition(request)),
+    )
+  }
+}
+
+object TrailUtils {
+  def normalisePillar(pillar: Option[Pillar]): String =
+    pillar match {
+      case Some(Pillar("arts")) => "culture"
+      case Some(Pillar(p))      => p
+      case None                 => "news"
+    }
+}
```

Comment:
This file is completely copy/paste except for method name `ToOnwardItem` => `ToTrail`